frontend/lib/app.tsx

```diff
@@ -198,8 +199,23 @@ export class AppWithoutRouter extends React.Component<AppPropsWithRouter, AppSta
         <Route path={Routes.login} exact component={LoginPage} />
         <Route path={Routes.adminLogin} exact component={LoginPage} />
         <Route path={Routes.logout} exact component={LogoutPage} />
-        <Route path={Routes.onboarding.prefix} component={LoadableOnboardingRoutes} />
+        <Route path={Routes.onboarding.prefix} render={() => (
+          <LoadableOnboardingRoutes
+            routes={Routes.onboarding}
+            toCancel={Routes.home}
+            toSuccess={Routes.loc.latestStep}
+            signupIntent={OnboardingInfoSignupIntent.LOC}
+          />
+        )} />
         <Route path={Routes.loc.prefix} component={LoadableLetterOfComplaintRoutes} />
+        <Route path={Routes.hp.onboarding.prefix} render={() => (
```

Comment:
It's actually really important that this route be before the next one, because the HP onboarding prefix begins with the HP prefix.